tools/src/main/java/org/apache/kafka/tools/ClientCompatibilityTest.java

```diff
@@ -430,8 +430,9 @@ public void testConsume(final long prodTimeMs) throws Throwable {
                 () -> log.info("offsetsForTime = {}", offsetsForTime.result));
             // Whether or not offsetsForTimes works, beginningOffsets and endOffsets
             // should work.
-            consumer.beginningOffsets(timestampsToSearch.keySet());
-            consumer.endOffsets(timestampsToSearch.keySet());
+            Map<TopicPartition, Long> beginningOffsets = consumer.beginningOffsets(timestampsToSearch.keySet());
+            Map<TopicPartition, Long> endingOffsets = consumer.endOffsets(timestampsToSearch.keySet());
+            log.trace("beginningOffsets: {}, endingOffsets: {}", beginningOffsets, endingOffsets);
```

Comment:
This is super annoying.

I started getting errors from SpotBugs because the offsets methods were called but the return value was being ignored. This is a very brute force way of silencing the checker. I could not find a clean way to ignore the warning.

I also don't know why this is suddenly being caught by SpotBugs. From its perspective, nothing has changed in the `KafkaConsumer` API, right?

Reply:
This is weird.

Reply:
It would be great if we could revert this change.